File: qc-core/src/main/java/com/eager/questioncloud/core/domain/user/CreateUser.java
package com.eager.questioncloud.core.domain.user;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class CreateUser {
    private String email;
    private String password;
    private String socialRegisterToken;
    private AccountType accountType;
    private String phone;
    private String name;
}

File: qc-core/src/main/java/com/eager/questioncloud/core/domain/user/dto/CreateUser.kt
package com.eager.questioncloud.core.domain.user.dto

import com.eager.questioncloud.core.domain.user.enums.AccountType

class CreateUser(
    val email: String,
    val password: String?,
    val socialRegisterToken: String?,
    val accountType: AccountType,
    val phone: String,
    val name: String,
)

